<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>669</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frameBlockchainInfo_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>451</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">border: transparent;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Block Height</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>29</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QLabel" name="label_24hblockchaininfo_1">
           <property name="styleSheet">
            <string notr="true">border: transparent;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Network Hashrate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frameBlockchainInfoData_3">
         <property name="styleSheet">
          <string notr="true">border: transparent;
; font-size: 15px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="0">
           <widget class="QLabel" name="label_CurrentBlock">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 15px</string>
            </property>
            <property name="text">
             <string>Current block:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_CurrentBlock_value_3">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 15px;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_Nethash_value_3">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_Nethash_3">
            <property name="styleSheet">
             <string notr="true">font-size: 15px;</string>
            </property>
            <property name="text">
             <string>Network MHs:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>99</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit HTH Partners and Donate Today!!!!!!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://hth.world/partners.html&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Partners</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_1">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Website</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://hth.world&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>HTH Coin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Worldwide A NJ Nonprofit Corporation</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://www.helpthehomelessworldwide.org&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Nonprofit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Explorer and Track Your HTH Coins</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://chainz.cryptoid.info/hth&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Explorer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visit AltMarkets Exchange and Trade Help The Homeless Coin Today!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://altmarkets.io/trading/hthbtc&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>AltMarkets</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Worldwide</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>280</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Coin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Exchanges</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_5">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visit Northern Exchange and Trade Help The Homeless Coin Today!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://nortexchange.com/exchange/?market=HTH_BTC&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>NorthernEX</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_blue_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>221</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelBalanceText">
           <property name="text">
            <string>Available:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelWatchAvailable_2">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Your current balance in watch-only addresses</string>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 15px</string>
             </property>
             <property name="text">
              <string notr="true">0.000 000 00 BTC</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelPendingText">
           <property name="text">
            <string>Pending:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="labelWatchPending_2">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Unconfirmed transactions to watch-only addresses</string>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 15px</string>
             </property>
             <property name="text">
              <string notr="true">0.000 000 00 BTC</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelImmatureText">
           <property name="text">
            <string>Immature:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="labelWatchImmature_2">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Mined balance in watch-only addresses that has not yet matured</string>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 15px</string>
             </property>
             <property name="text">
              <string notr="true">0.000 000 00 BTC</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelTotalText_2">
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelWatchTotal_2">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Current total balance in watch-only addresses</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00 BTC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: #debf12;</string>
      </property>
      <property name="text">
       <string>HTH Information</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>639</x>
        <y>79</y>
        <width>331</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelBalance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Your current spendable balance</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 35px</string>
      </property>
      <property name="text">
       <string notr="true">0.000 000 00 BTC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Balances</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelWalletStatus">
           <property name="toolTip">
            <string>The displayed information may be out of date. Your wallet automatically synchronizes with the Dash network after a connection is established, but this process has not completed yet.</string>
           </property>
           <property name="text">
            <string notr="true">(out of sync)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelTotalText">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px</string>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
